src/rollback_to_stable/rts_btree.c

```diff
@@ -146,12 +151,16 @@ __rts_btree_abort_insert_list(WT_SESSION_IMPL *session, WT_PAGE *page, WT_INSERT
                 WT_ERR(__wt_vpack_uint(&memp, 0, recno));
                 key->size = WT_PTRDIFF(memp, key->data);
             }
+            __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_4,
+              WT_RTS_VERB_TAG_UPDATE_ABORT
+              "attempting to abort update on the insert list with durable_ts:%" PRIu64,
```

Comment:
This one should be `durable_ts=` for consistency, there's another one below as well.